src/LRSLib.jl

```diff
@@ -2,6 +2,7 @@ module LRSLib
 
 using BinDeps
 using Polyhedra
+using Markdown
```

Comment:
Why is this needed ? What does it do ?

Reply:
This is needed for the docstring with `@doc doc` for `lpsolve` (which uses LaTeX commands).